tests/apollo/util/bft_debug_tool.py

```diff
@@ -0,0 +1,144 @@
+# Concord
+#
+# Copyright (c) 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Version 2018-2022

Reply:
I'm not sure about it, if you modify file you can do it. This is a new file, should get the current year.
I'll fix it anyway, it doesnt matter much.